app/src/main/java/com/worldsnas/daggerfeatures/fragments/FourthFragment.kt

```diff
@@ -2,5 +2,6 @@ package com.worldsnas.daggerfeatures.fragments
 
 import androidx.fragment.app.Fragment
 
+//@AndroidEntryPoint
```

Comment:
why is this commented out?

Reply:
in this fragments there is no dependency (@inject) , so we dont need to annotate.
After complete this class if there was any dependency we will comment it out.

Reply:
ahh get it now.
please remove the comment now (to have the code cleaner)
and then we add it when the time comes

Reply:
removed and pushed